crates/re_viewer/src/ui/time_panel/mod.rs

```diff
@@ -589,15 +628,44 @@ fn paint_streams_guide_line(
 fn top_row_ui(ctx: &mut ViewerContext<'_>, ui: &mut egui::Ui) {
     ui.spacing_mut().item_spacing.x = 18.0; // from figma
 
-    ctx.rec_cfg
-        .time_ctrl
-        .time_control_ui(ctx.re_ui, ctx.log_db.times_per_timeline(), ui);
+    if ui.max_rect().width() < 600.0 {
```

Comment:
nit: most of this stuff could still be in `time_control_ui.rs`. This mod.rs is a bit crowded.

Reply:
Not easily. The "draggable timeline" element of it is not at all part of `time_control_ui.rs` at the moment.